Sources/MountebankSwift/Common/HTTPMethod.swift

```diff
@@ -12,3 +12,9 @@ public enum HTTPMethod: String, Codable, Equatable {
     case trace = "TRACE"
     case patch = "PATCH"
 }
+
+extension HTTPMethod: Recreatable {
```

Comment:
maybe move to separate  file?

Reply:
I considered this, but kept all the `Recreatable` conformance in the same files because it's important that they are kept in sync with the `init`. So I would keep them as close to that `init` as possible. What do you think?